File: snippets/php/remove_bulk_tags.php
require_once('/path/to/MailchimpMarketing/vendor/autoload.php');

$mailchimp = new MailchimpMarketing\ApiClient();

$mailchimp->setConfig([
    'apiKey' => 'YOUR_API_KEY',
    'server' => 'YOUR_SERVER_PREFIX'
]);

$list_id = "YOUR_LIST_ID";
$tag_id = YOUR_TAG_ID;

$members_to_remove = new MailchimpMarketing\Model\MembersToAddremoveTofromAStaticSegment();
$membersToAdd->setMembersToRemove(["dolly.parton@example.com", "rihanna@example.com"]);

try {
    $response = $mailchimp->lists->batchSegmentMembers($members_to_remove, $list_id, $tag_id);
    echo "Successfully untagged {$response->getTotalRemoved()} contacts";
} catch (MailchimpMarketing\ApiException $e) {
    echo $e->getMessage();
}

File: snippets/python/remove_bulk_tags.py
import mailchimp_marketing as MailchimpMarketing
from mailchimp_marketing.api_client import ApiClientError

API_KEY = "YOUR_API_KEY"
SERVER_PREFIX = "YOUR_SERVER_PREFIX"

LIST_ID = "YOUR_LIST_ID"
TAG_ID = "YOUR_TAG_ID"

client = MailchimpMarketing.Client()
client.setConfig(API_KEY, SERVER_PREFIX)
try:
    response = client.lists.batch_segment_members({
        "members_to_remove": [
            "john.smith@example.com",
            "brad.hudson@example.com",
        ]
    }, LIST_ID, TAG_ID)
    print("client.ping.get() response: {}".format(response))
except ApiClientError as error:
    print("An exception occurred: {}".format(error.text))